cypress/integration/shortcuts.spec.ts

```diff
@@ -0,0 +1,41 @@
+import * as commands from "../support/commands";
+
+context("documented shortcuts", () => {
+  before(() => {
+    commands.resetProxyState();
+    commands.onboardUser("cloudhead");
+    cy.visit("./public/index.html");
+  });
+
+  it("opens and closes the keyboard shortcuts help", () => {
+    cy.get("body").type("?");
+    commands.pick("hotkey-modal").should("exist");
+    // Closing the modal
+    cy.get("body").type("{esc}");
+    commands.pick("hotkey-modal").should("not.exist");
+  });
+
+  it("opens and closes the project creation", () => {
+    cy.get("body").type("{meta}n");
```

Comment:
```suggestion
    cy.get("body").type("{meta}");
```

?

Reply:
dismiss this comment, I did not match it with the context because of `{meta}`. Looking good :ok_hand: